setup.cfg

```diff
@@ -4,8 +4,20 @@ test=pytest
 [flake8]
 max-line-length = 120
 # W503 raises a warning when there is a line break before a binary operator.
-# This is best practice according to PEP 8 and the rule should be ignored. 
+# This is best practice according to PEP 8 and the rule should be ignored.
 #
 # https://www.flake8rules.com/rules/W503.html
 # https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator
 ignore = W503
+
+[mypy]
```

Comment:
These are a fairly common set of flags to enable, i.e., they have been hugely valuable for projects like Apache Superset as configured [here](https://github.com/apache/superset/blob/f37fc1a7f0297ff3348221ed784a1034d2cf44e9/setup.cfg#L37-L44).